Project1/Index.html

```diff
@@ -5,28 +5,31 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Oswald:500" rel="stylesheet">
```

Comment:
You can use 2 Google web fonts in one stylesheet; you can replace lines 9 and 10 with this single line:


`<link href="https://fonts.googleapis.com/css?family=Libre+Franklin|Oswald:500" rel="stylesheet">`